app/src/main/res/layout/screen_edit_patient.xml

```diff
@@ -196,7 +197,6 @@
         android:text="@string/patientedit_add_button"
         android:textAlignment="viewStart"
         android:textAppearance="?attr/textAppearanceButton"
-        android:visibility="gone"
```

Comment:
We need bp passport button to be gone by default, I have a story for this [here](https://app.shortcut.com/simpledotorg/story/6763/show-bp-passport-text-when-bp-passport-is-null-as-well) which will update this when feature flag is true for this feature. Can you undo this, please?

Reply:
oh good catch. I've already made this change tho in the feature flag PR. So we don't need to handle [this](https://app.shortcut.com/simpledotorg/story/6393/move-adding-health-ids-from-edit-patient-behind-a-feature-flag) anymore.

Reply:
We'll still need to remove this from this PR, if you've added in that and that PR is merged in, here you're removing it again

Reply:
yeah, I've changed that and pushed